src/olympia/devhub/forms.py

```diff
@@ -1433,4 +1434,8 @@ def clean_origin(self):
             or parsed.fragment
         ):
             raise forms.ValidationError(error_message)
+        if DeniedInstallOrigin.find_denied_origins([value]):
+            raise forms.ValidationError(
+                str(DeniedInstallOrigin.ERROR_MESSAGE).format(origin=value)
```

Comment:
I guess that doesn't get cast into a string anyway along the way?

Reply:
Looks like `format()` does it actually, so the explicit cast is not needed. I'll remove.